src/prompts/prompts.scss

```diff
@@ -0,0 +1,8 @@
+@import "./styles/variables";
+@import "./styles/anims";
+@import "./styles/tooltip";
+@import "./styles/toast";
+@import "./styles/prompts";
```

Comment:
Question on file naming, the files are underscore prefixed, which sass says means they're a 'partial', and therefore should include with `@use`.

I'm presuming this is working fine, just curious if we're missing anything here as I'm not familiar with partials in sass.

Reply:
Ahh I did have @use (as knew @import would be deprecated) however did encounter an issue with it not bundling the styles at all (the @import approach with partials was what I was more familiar with) - can take another look however one the deck is cleared 👍

Reply:
OK some issue with @use remains, will revisit later as @import is working just now.